src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -342,13 +342,15 @@ Computes the water flux given the absolute potential (pressure/(ρg))
 and the conductivity along
 the flow path between these two points.
 
-We currently assuming an arithmetic
-mean for mean K_sat between the two points (Bonan, 2019; Zhu, 2008)
-to take into account the change in K_sat halfway
-between z1 and z2; this is incorrect for compartments of differing sizes.
-"""
-function flux(z1, z2, ψ1, ψ2, K1, K2)
-    flux = -(K1 + K2) / 2 * ((ψ2 - ψ1) / (z2 - z1) + 1)
+We currently assuming an geometric
+mean for mean K_sat between the two points. Previously,
+we used the arithmetic mean (Bonan, 2019; Zhu, 2008),
+but then when the soil K was very low, root extraction would
+continue. This should be modified for compartments of differing sizes.
```

Comment:
what do you mean by "This should be modified for compartments of differing sizes."? what should be modified?

Reply:
I added a comment for what CLM does, but will double check.

Reply:
Ok, I fleshed out the comment more. This is something we probably should do. But it would require introducing additional parameters (root length density, root biomass). I believe the PlantHydraulics code will need some other improvements too, and I think we could tackle it then.